sdk/src/assertions/box_hash.rs

```diff
@@ -301,18 +301,20 @@ impl AssertionBase for BoxHash {
     }
 }
 
+#[cfg(feature = "file_io")]
 #[cfg(test)]
 mod tests {
     #![allow(clippy::unwrap_used)]
 
     use super::*;
+    #[cfg(test)]
     use crate::{jumbf_io::get_assetio_handler_from_path, utils::test::fixture_path};
 
     #[test]
     fn test_hash_verify_jpg() {
         let ap = fixture_path("CA.jpg");
 
-        let bhp = get_assetio_handler_from_path(&ap)
+        let bhp = crate::jumbf_io::get_assetio_handler_from_path(&ap)
```

Comment:
Do you plan to expose the assert handlers or is this just a test/work-in-progress?

Reply:
It is referenced this way because it isn't exposed. My plan was not even expose jumbf_io. But I may allow a couple of functions under a feature flag.